projects/test_project/rules.mk

```diff
@@ -0,0 +1,5 @@
+# Specify the main file (relative to this file if not in this folder)
+MAIN := main.c
+
+# Specify the device library you want to include
+APP_DEPS := ms-lib
```

Comment:
Unrelated note for future reference: add HEADERS variable to these types of files.